gemfiles/rails_main.gemfile

```diff
@@ -1,6 +1,6 @@
 source "https://rubygems.org"
 
 gem "activerecord", github: 'rails/rails', branch: 'main', :require => "active_record"
-gem "sqlite3", "~> 1.4"
+gem "sqlite3", "~> 2.0"
```

Comment:
Should we instead of pinning to 2.x leave it more open, so that we don’t need to update this Gemfile when sqlite3 3.0 comes out? Rails doesn’t specify an upper limit now.